website/docs/migrating-from-docker/verifying-your-tools-are-using-podman.md

```diff
@@ -55,21 +55,21 @@ When you have configured your host to use Podman rather then Docker, consider ve
      <TabItem value="win" label="Windows">
 
    ```shell-session
-   $ CONTAINER_HOST=npipe:////./pipe/docker_engine podman ps
+   $ CONTAINER_HOST=npipe:////./pipe/docker_engine; podman ps
```

Comment:
```suggestion
   $ set CONTAINER_HOST=npipe:////./pipe/docker_engin && podman ps
```

Reply:
this breaks examples similar to how we pass in DOCKER_HOST for compose examples, I don't believe we should implement this change.